plugins/hls-tactics-plugin/COMMANDS.md

```diff
@@ -0,0 +1,448 @@
+# Wingman Metaprogram Command Reference
+
+## application
+
+arguments: none.
+non-deterministic.
```

Comment:
Maybe better to have two spaces after `none.` (so that there will be two separate lines)?
I mean like
```suggestion
arguments: none.  
non-deterministic.
```